tests/remotes/gs.py

```diff
@@ -25,27 +25,13 @@
 class GCP(Base, CloudURLInfo):
     @staticmethod
     def should_test():
-        from subprocess import CalledProcessError, check_output
-
         do_test = env2bool("DVC_TEST_GCP", undefined=None)
         if do_test is not None:
             return do_test
 
         if not os.path.exists(TEST_GCP_CREDS_FILE):
             return False
 
-        try:
-            check_output(
-                [
-                    "gcloud",
-                    "auth",
-                    "activate-service-account",
-                    "--key-file",
-                    TEST_GCP_CREDS_FILE,
-                ]
-            )
-        except (CalledProcessError, OSError):
-            return False
```

Comment:
Had to get rid of this to make gs tests less flaky. This is an old leftover that is no longer needed after test_data_cloud CLI refactoring done a few months back, where we started properly using config.